src/Simplify.elm

```diff
@@ -932,22 +929,40 @@ fromProjectToModule =
                 moduleExposedVariantTypes : Exposed
                 moduleExposedVariantTypes =
                     moduleExposingContext fullAst.moduleDefinition
+
+                imports : ImportLookup
+                imports =
+                    List.foldl
+                        (\import_ importLookup ->
+                            let
+                                importInfo : { moduleName : ModuleName, exposed : Exposed, alias : Maybe ModuleName }
+                                importInfo =
+                                    importContext import_
+                            in
+                            insertImport importInfo.moduleName { alias = importInfo.alias, exposed = importInfo.exposed } importLookup
+                        )
+                        Dict.empty
```

Comment:
>if someone has an import import Platform.Cmd exposing (none), then the Cmd alias will currently be forgotten about as well as the Cmd type.

Oh, what? That's so weird because for example
> import Basics

you can still have unqualified `identity` etc. Maybe a special case with `exposing (..)` or `Basics`?

Reply:
Sorry, I meant to say that with the current implementation (but I might be wrong, would probably warrant a test), if we see a `import Platform.Cmd exposing (none)`, then `Cmd.none` is not going to be known, because the prelude import got overriden because of the `Dict.union`. The `insertImport` approach handles that correctly.

(when I say "current", I meant the one from when I did the review, because I see you applied my suggestion now).